libexec/tachyon-config.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 # A developer option to prepend Tachyon jars before TACHYON_CLASSPATH jars
 if [ -n "$TACHYON_PREPEND_TACHYON_CLASSES" ]; then
-  export CLASSPATH="$TACHYON_CONF_DIR/:$TACHYON_JARS:$TACHYON_CLASSPATH"
+  export CLASSPATH="$TACHYON_CONF_DIR/:$TACHYON_JARS:$TACHYON_CLASSPATH:$CLASSPATH"
```

Comment:
Moved the adding to `tachyon-application-master.sh`, and added a comment on why it's needed.